### PR TITLE
add process.end

### PR DESCRIPTION
**Pull request by ferullo**
## Change Summary

Add `process.end` from ECS into Endpoint process documents.

### Sample values

See blow

Sample document:
Heavily redacted/snipped example:

```json
{
    "message": "Endpoint process event",
    "process": {
        "end": "2022-07-18T21:05:19.9419692Z"
    }
}
```

## Release Target

8.4

## Q/A

N/A

### For mapping changes:

- [x] I ran `make` after making the schema changes, and committed all changes
- [ ] If these field(s) are "exception"-able, I made a companion PR to Kibana adding it (see [Readme](https://github.com/elastic/endpoint-package#exceptionable))
- [ ] If this is a `metadata` change, I also updated both transform destination schemas to match

### For Transform changes:

- [ ] The new transform successfully starts in Kibana
- [ ] The corresponding transform destination schema was updated if necessary

### CODE REVIEW

**Comment by ferullo** [comment]
👍 done

File: package/endpoint/data_stream/process/sample_event.json [hunk 1]
```diff
@@ -159,4 +160,4 @@
         "name": "SYSTEM",
         "id": "S-1-5-18"
     }
-}
\ No newline at end of file
+}
```
Comment (pzl) [inline package/endpoint/data_stream/process/sample_event.json:163]: I vaguely recall the linter being very particular about not liking final newlines. Unfortunately I don't think I can relax its rules. We will see if CI complains

Comment (pzl) [reply package/endpoint/data_stream/process/sample_event.json:163]: yep, 

```
[2022-07-20T16:39:57.255Z] + git diff-index --exit-code HEAD --
[2022-07-20T16:39:57.255Z] :100644 100644 177a6e95dde05a0e739866f032964d3127fa4ebd 0000000000000000000000000000000000000000 M	package/endpoint/data_stream/process/sample_event.json
script returned exit code 1
```

after a `format` is run, it seems to standardize no-ending-newline and then we detect a git diff on that.

For the future I will see if we can relax that, but for now the fix here is to delete trailing \n

Comment (ferullo) [reply package/endpoint/data_stream/process/sample_event.json]: Thanks for the heads up. It should be fixed now.

**Review by Trinity2019 — APPROVED** [review]

**Pull request merged**